lib/entities/default_settings_migration.dart

```diff
@@ -276,7 +276,9 @@ Future<void> defaultSettingsMigration(
             oldUri: 'cakewallet.com',
           );
           break;
-
+        case 45:
+          await updateETHNodesWithNowNodes(sharedPreferences: sharedPreferences, nodes: nodes);
+         await updatePolygonNodesWithNowNodes(sharedPreferences: sharedPreferences, nodes: nodes);
```

Comment:
make a generic function like the one above in case 44

Reply:
Alright